tests/node/adapter.test.ts

```diff
@@ -50,23 +138,39 @@ suite("create", () => {
     expect(actual.userId).toBe(user.id)
   })
 
+  // https://github.com/octet-stream/better-auth-mikro-orm/issues/18
+  test("with a reference where the referenced entity is not in the identity map (issue #18)", async () => {
```

Comment:
This shouldn't be there if you want to reference the issue for this test case. I already have `issues` suite just for this: https://github.com/octet-stream/better-auth-mikro-orm/blob/011376ceb0de22dd26529c7a0546e557e385a03e/tests/node/adapter.test.ts#L338